ios/testmanagerd/xcuitestrunner.go

```diff
@@ -243,7 +243,7 @@ func RunXCUITest(bundleID string, testRunnerBundleID string, xctestConfigName st
 		xctestConfigName = info.targetAppBundleName + "UITests.xctest"
 	}
 
-	return RunXCUIWithBundleIdsCtx(context.TODO(), bundleID, testRunnerBundleID, xctestConfigName, device, nil, env, testListener)
+	return RunXCUIWithBundleIdsCtx(context.TODO(), bundleID, testRunnerBundleID, xctestConfigName, device, nil, env, testsToRun, testsToSkip, testListener)
 }
 
 func RunXCUIWithBundleIdsCtx(
```

Comment:
As a next step we probably should put those args into a config struct so that it's easier to use.

Reply:
Agree. Wanna see it in this MR?